<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorLIADialog</class>
 <widget class="QDialog" name="SensorLIADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QCheckBox" name="outputBox">
     <property name="text">
      <string>Enable output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="liaSettingsButton">
     <property name="text">
      <string>Edit LIA-engine settings</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="gainLabel">
     <property name="text">
      <string>PGA gain</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="gainBox">
     <item>
      <property name="text">
       <string>Auto (defaults to 1)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="gPeriodLabel">
     <property name="text">
      <string>Measurement period</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="gPeriodBox">
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string notr="true">s</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>6553.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="confirmBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="gPeriodButton">
     <property name="text">
      <string>Edit global measurement policy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmBox</sender>
   <signal>accepted()</signal>
   <receiver>SensorLIADialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmBox</sender>
   <signal>rejected()</signal>
   <receiver>SensorLIADialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
